api/src/test/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/RepeatVectorLayerTest.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 JetBrains s.r.o. and Kotlin Deep Learning project contributors. All Rights Reserved.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlinx.dl.api.core.layer
+
+import org.jetbrains.kotlinx.dl.api.core.KGraph
+import org.jetbrains.kotlinx.dl.api.core.layer.reshaping.RepeatVector
+import org.jetbrains.kotlinx.dl.api.core.shape.toIntArray
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.tensorflow.Graph
+import org.tensorflow.Output
+import org.tensorflow.op.Ops
+
+/**
+ * A test for the [RepeatVector] layer.
+ *
+ * @author Stan van der Bend
+ */
+internal class RepeatVectorLayerTest {
+
+    @Test
+    fun `test output shape`(){
+        val layer = RepeatVector(n = 2)
+        val x = Array(10) { FloatArray(10) { 1F } }
+        val y = layer(x)
+        Assertions.assertArrayEquals(intArrayOf(10, layer.n, 10), y.shape().toIntArray())
+    }
+
+    @Test
+    fun `test repetition output`(){
+        val layer = RepeatVector(n = 2)
+        val x = Array(3) { FloatArray(3) { it.toFloat() } }
+        val y = layer(x)
+        val actual = y.tensor().copyTo(Array(3) { Array(layer.n) { FloatArray(3) } })
+        val expected = arrayOf(
+            arrayOf(floatArrayOf(0F, 1F, 2F), floatArrayOf(0F, 1F, 2F)),
+            arrayOf(floatArrayOf(0F, 1F, 2F), floatArrayOf(0F, 1F, 2F)),
+            arrayOf(floatArrayOf(0F, 1F, 2F), floatArrayOf(0F, 1F, 2F))
+        )
+        Assertions.assertArrayEquals(expected, actual)
+    }
+
+    private operator fun RepeatVector.invoke(input : Array<FloatArray>) : Output<Float> {
```

Comment:
Very good idea from my perspective but maybe open a separate issue for this or join [this discussion](https://github.com/JetBrains/KotlinDL/discussions/132) with your improvement to test readability and some proposal. Btw it should be done for the `Layer` and then reused in all tests in the way you did this.

Reply:
btw clean this code a little bit (I don't see the need to refer to `Shape` as `org.tensorflow.Shape`) and let's see that the block with single value `{ return value }` can be reduced to `= value` (especially when you use the `.let { }`)